framework/doc/content/source/mfem/auxkernels/MFEMInnerProductAux.md

```diff
@@ -0,0 +1,33 @@
+# MFEMInnerProductAux
+
+!if! function=hasCapability('mfem')
+
+## Summary
+
+!syntax description /AuxKernels/MFEMInnerProductAux
```

Comment:
we generally omit this from the `.md` page since documentation such as in your `Overview` typically is a superset of the class description

Reply:
Do you mean the `## Summary` heading itself? Because the syntax description is definitely a common feature and is what you get in the syntax index. As for the heading, it's missing in like 10 files, I can add it (I just checked, it was actually already on my todo list).